### PR TITLE
:sparkles: Proxy starterpack requests through ozone and include takedowns

### PR DESCRIPTION
**Pull request by foysalit**

### CODE REVIEW

File: packages/dev-env/src/seed/basic.ts [hunk 1]
```diff
@@ -155,6 +155,8 @@ export default async (
     })
   }
 
+  await sc.network.processAll()
+
```
Comment (devinivy) [inline packages/dev-env/src/seed/basic.ts:159]: All the other seeds let the caller perform the final `processAll()`— does it cause any issues if we stick with that for now?

Comment (devinivy) [reply packages/dev-env/src/seed/basic.ts:159]: I see in the new test suite it works the same way:
```ts
    await basicSeed(sc)
    await network.processAll()
```

**Review by devinivy — APPROVED** [review]
Looks great, thanks!

File: packages/dev-env/src/seed/basic.ts (hunk 1 above)
Comment (foysalit) [reply packages/dev-env/src/seed/basic.ts]: nope, don't think there's any issue with that. just cleaned up!

**Pull request merged**